src/logo-system/logo-elements/index.md

```diff
@@ -122,21 +122,24 @@ If it’s too small, it can lose detail and be harder for some users to read or
 
     {% gridCell %}
 
-    ![The wordmark for "GOV.UK" with an arrow indicating its minimum width.](./wordmark-min-width.svg)
-    Minimum size:</br>
+    ![](./wordmark-min-width.svg)
+
+    Wordmark minimum width:</br>
```

Comment:
I initially used "Minimum width of wordmark" as that is better English. But that went into two lines in desktop view.